### ISSUE TITLE
[BUG] "Cannot create so many PeerConnections" on Chrome after 500 accepted calls

### ISSUE THREAD

**Issue by asmeikal** [twilio/twilio-voice.js#145]
- [x] I have verified that the issue occurs with the latest twilio.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-voice.js/blob/master/CHANGELOG.md).
 - [x] I reviewed the [Common Issues](https://github.com/twilio/twilio-voice.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [x] I verified that the [Quickstart](https://github.com/TwilioDevEd/voice-javascript-sdk-quickstart-node) application works in my environment.
 - [x] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii) or sensitive account information (API keys, credentials, etc.) when reporting this issue.

**Code to reproduce the issue:**

An example is available here: https://github.com/asmeikal/voice-javascript-sdk-quickstart-node

After starting up the device, click on the "Start loop" button. The script will connect and then disconnect calls in a loop, keeping a counter.

**Expected behavior:**

An unlimited number of calls can be made.

**Actual behavior:**

As soon as the counter reaches 500, no more calls can be made. The stack trace of the error is the following:

```
twilio.min.js:1 Uncaught (in promise) DOMException: Failed to construct 'RTCPeerConnection': Cannot create so many PeerConnections
    at RTCPC.create (http://localhost:3000/twilio.min.js:1:184344)
    at PeerConnection._setupPeerConnection (http://localhost:3000/twilio.min.js:1:171217)
    at PeerConnection._initializeMediaStream (http://localhost:3000/twilio.min.js:1:173968)
    at PeerConnection.makeOutgoingCall (http://localhost:3000/twilio.min.js:1:176510)
    at connect (http://localhost:3000/twilio.min.js:1:36367)
    at http://localhost:3000/twilio.min.js:1:36907
```

The root of the problem is this Chromium bug, that is still open: https://bugs.chromium.org/p/chromium/issues/detail?id=825576
RTCPeerConnections are not garbage collected quickly enough by Chrome, but workarounds are available, as shown in this comment: https://bugs.chromium.org/p/chromium/issues/detail?id=825576#c40
Forcing garbage collection through the workaround in the comment, or by relying directly on the garbage collector in Electron, reclaims part of the RTCPeerConnections, and other RTCPeerConnections can be made.

However, the workarounds do not work when using the Twilio JS Voice SDK, even after waiting up to an hour and after recreating the Twilio device.
This may be caused by references to the RTCPeerConnection being leaked by the Twilio JS Voice SDK, as pointed out in some of the comments: https://bugs.chromium.org/p/chromium/issues/detail?id=825576#c39
This prevents even a manual trigger of the garbage collection from cleaning up the closed RTCPeerConnections.

We detected this problem in our application, where call center operators usually reach the 500 calls after roughly 6 hours.

**Software versions:**

 - [x] Browser(s): Chromium 108, Electron 18 
 - [x] Operating System: Windows 10, 11
 - [x] twilio.js: 2.1.2, 2.2.0
 - [ ] Third-party libraries (e.g., Angular, React, etc.):

**Comment by asmeikal** [comment]
As requested by the Twilio support team, here is a Call SID for which we detected the problem: CA72b33d8da7bdd27d331f4f6283910143.

Twilio logs for this call only show the following log:

```
Canceled: CA72b33d8da7bdd27d331f4f6283910143
```

This call was received by our application after the 500 calls. What happened was the following:

1. The Device received the "incoming" event
2. We programmatically called the "accept" method on the Call object inside the "incoming" event
3. We detected the error shown in the first message
4. After 5-10 seconds, the Call object received the "cancel" event
